view/adminhtml/templates/origin_order_data.phtml

```diff
@@ -4,11 +4,14 @@
  */
 ?>
 <?php if ($block->getExternalId()): ?>
-    <div class="admin__page-section-title">
-        <span class="title">
+    <div class="admin__page-section-title"> <span class="title">
```

Comment:
It was more redeable when span was in the new line

Reply:
Done